user_docs/en/userGuide.t2t

```diff
@@ -1570,11 +1570,17 @@ Following are the key assignments for this display with NVDA.
 Please see the display's documentation for descriptions of where these keys can be found.
 %kc:beginInclude
 || Name | Key |
-| Scroll braille display back | t1 |
+| Scroll braille display back | t1 or etouch1 |
 | Move braille display to previous line | t2 |
+| Move to focus object | t3 |

```

Comment:
We don't mention "object" elsewhere for this command. Also, it moves to the cursor, not just to the object. I suggest this be changed to "Move to current focus".